.github/renovate.json5

```diff
@@ -18,6 +18,10 @@
     'type: process',
   ],
   packageRules: [
+    {
+      matchFileNames: ["samples/**"],
+      groupName: "samples-deps",
```

Comment:
nit: "Samples". This will have title: "chore(deps): update samples"

Reply:
```suggestion
      groupName: "Samples",
```